fili-generic-example/README.md

```diff
@@ -58,26 +67,37 @@ Here are some sample queries that you can run to verify your server:
 
 ### Specific to Wikipedia data
```

Comment:
Thank you for your comments! I updated the commands and added some links, and I pushed the (fixed) image to Dockerhub. I also moved the Docker instructions from the generic example to the wikipedia example. Please let me know if it's still not working, or if there's anything else I should change.